test/parallel_exec.jl

```diff
@@ -15,6 +15,20 @@ end
 
 addprocs(3; exeflags=`$cov_flag $inline_flag --check-bounds=yes --depwarn=error`)
 
+println("Testing fake connection attempts to workers. Please ignore error messages regarding invalid connection credentials.")
+all_w = workers()
+# Test sending fake data to workers. The worker processes will print an
+# error message but should not terminate.
+for w in Base.PGRP.workers
+    if isa(w, Base.Worker)
+        s = connect(get(w.config.host), get(w.config.port))
+        write(s, randstring(32))
+    end
+end
+@test workers() == all_w
+@test all([p == remotecall_fetch(myid, p) for p in all_w])
+
+
```

Comment:
Nevermind. The messages look pretty ugly in the CI logs. Will change text and move test under `JULIA_TESTFULL`.